packages/dds/tree/src/test/shared-tree/fuzz/operationTypes.ts

```diff
@@ -8,13 +8,27 @@ import type { DownPath } from "../../../feature-libraries/index.js";
 
 export type Operation = TreeOperation | Synchronize;
 
-export type TreeOperation = TreeEdit | TransactionBoundary | UndoRedo | SchemaChange;
+export type TreeOperation =
+	| TreeEdit
+	| TransactionBoundary
+	| UndoRedo
+	| SchemaChange
+	| Constraint;
 
 export interface TreeEdit {
 	type: "treeEdit";
 	edit: FieldEdit;
 }
 
+// Currently only node constraints are supported, but more constraint types may be added in the future.
+export interface Constraint {
+	type: "constraint";
+	content: NodeConstraint;
+}
+export interface NodeConstraint {
+	type: "nodeConstraint";
+	content: undefined | DownPath;
```

Comment:
I think a name like "path", "target", or "node" would make more sense than "content."

What does it mean if the path is undefined?